shared/wallets/sep7-confirm/index.tsx

```diff
@@ -223,7 +223,7 @@ const SEP7Confirm = (props: Props) => (
           type="Success"
           onClick={
             props.operation === 'pay'
-              ? () => props.onAcceptPay(props.amount || props.userAmount)
+              ? () => props.onAcceptPay(props.amount || props.userAmount || '' /* TODO is this ok?*/)
```

Comment:
@keybase/picnicsquad please validate

Reply:
Would it help if SEP7Confirm itself renders a spinner on checking loading as the first thing it does?  We should catch SEP7ConfirmInfo becoming null earlier than here.